apps/server/src/modules/lesson/repo/lesson.repo.integration.spec.ts

```diff
@@ -202,35 +202,89 @@ describe('LessonRepo', () => {
 		});
 	});
 
-	describe('updateLessons', () => {
-		it('should update Lessons without deleted user', async () => {
-			// Arrange
-			const userId = new ObjectId();
-			const contentExample: ComponentProperties = {
-				title: 'title',
+	describe('removeUserReference', () => {
+		const setup = async () => {
+			const userId1 = new ObjectId();
+			const userId2 = new ObjectId();
```

Comment:
Could we use `const userId1 = new ObjectId().toHexString()` and work with `EntityId` instead of `ObjectId`?